app/components/location_restriction_component.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# This is only used by the PdfComponent
+class LocationRestrictionComponent < ViewComponent::Base
+  def initialize(location = 'site visitors to the Stanford Libraries')
```

Comment:
We never initialize this without an argument, do we? Why do we need a default value?